test/astarte_core/triggers/simple_events_test.exs

```diff
@@ -0,0 +1,415 @@
+defmodule Astarte.Core.SimpleEventsTest do
+  use ExUnit.Case
+
+  describe "simple events protobuf encoding" do
```

Comment:
Specify that this tests the compatibility with the old library, otherwise it's not obvious why this tests are here